designs/unice/src/gusset.mjs

```diff
@@ -37,22 +37,42 @@ export const gusset = {
     var yScaleDoubleLayer
     yScaleDoubleLayer = (1 + store.get('yScale')) / 2 // double layer of fabric stretches half as much
 
+    let actualGussetLength
+    actualGussetLength = measurements.seat * options.gussetLength * yScaleDoubleLayer
+
     // Create points
     points.frontGussetLeft = new Point(store.get('frontGussetLeft').x, 0)
-    points.backGussetLeft = new Point(
-      store.get('backGussetLeft').x,
-      measurements.seat * options.gussetLength * yScaleDoubleLayer
-    )
+    points.backGussetLeft = new Point(store.get('backGussetLeft').x, actualGussetLength)
     points.frontGussetRight = new Point(store.get('frontGussetRight').x, 0)
-    points.backGussetRight = new Point(
-      store.get('backGussetRight').x,
-      measurements.seat * options.gussetLength * yScaleDoubleLayer
-    )
+    points.backGussetRight = new Point(store.get('backGussetRight').x, actualGussetLength)
+
+    let gussetWidthFront
+    gussetWidthFront = points.frontGussetRight.x - points.frontGussetLeft.x
+
+    let backExposureThreshold = 0.45 // where the gusset becomes triangular
 
     // Create control points
-    points.gussetCp1 = points.frontGussetLeft
-      .shiftFractionTowards(points.backGussetLeft, 0.5)
-      .shift(180, points.frontGussetRight.x / -15)
+    if (options.backExposure > backExposureThreshold) {
+      // gusset is roughly triangular
+      points.gussetCp1 = points.frontGussetLeft
+        .shiftFractionTowards(points.backGussetLeft, 0.3)
+        .shift(0, (0.1 * gussetWidthFront) / options.backExposure)
+    } else {
+      // ensure that gusset has a 'waist' at ~25% of the way from front to back
+      let horShift
+      horShift =
+        actualGussetLength / gussetWidthFront > 1.5
+          ? (0.1 * gussetWidthFront) / backExposureThreshold
+          : (0.1 * actualGussetLength) / (1.5 * backExposureThreshold)
+
+      points.gussetCp1 = points.frontGussetLeft
+        .shift(270, 0.3 * actualGussetLength) // shift to same height as for high back exposure
+        .shift(0, horShift)
+      points.gussetCp1 = points.gussetCp1.shiftFractionTowards(
+        points.frontGussetLeft,
+        Math.max(0, options.backExposure) - backExposureThreshold
+      )
+    }
 
     // Flip points to right side
     points.gussetCp2 = points.gussetCp1.flipX(store.get('frontGussetMid'))
```

Comment:
I note that this change, later down below in the code, would put the dimension inside the part, so it doesn't intersect the leg seam:
```diff
       macro('vd', {
         from: points.frontGussetRight,
         to: points.backGussetRight,
-        x: points.frontGussetRight.x + sa + 15,
+        x: points.frontGussetRight.x - sa - 15,
       })
     }
```

And, this would put the dimension outside the part entirely:
```diff
       macro('vd', {
         from: points.frontGussetRight,
         to: points.backGussetRight,
-        x: points.frontGussetRight.x + sa + 15,
+        x: points.backGussetRight.x + sa + 15,
       })
     }
```

(Just suggestions, please feel free to ignore.)